README.md

```diff
@@ -188,6 +188,13 @@ automatically.
 Called when an item is removed from the cache and should be disposed.  Set
 this on the constructor options.
 
+### `setTimer`
```

Comment:
Should this even be listed here?

Reply:
If something is exposed at all, I prefer to go ahead and document it as internal, so that if someone sticks their fingers in the machine and gets cut, at least they were properly warned.

Reply:
Just verified, can make it a #private method, but only in node 14 and higher.  Node 12 only supported private properties, not methods.  Will make that change and bump engines and semver major after landing this.